Plutarch/Api/V1.hs

```diff
@@ -71,24 +69,21 @@ import Plutarch.DataRepr (
   PIsDataRepr,
   PIsDataReprInstances (PIsDataReprInstances),
   PIsDataReprRepr,
-  PVia (..),
   pmatchDataRepr,
   pmatchRepr,
  )
 import Plutarch.Integer (PInteger, PIntegral)
-import Plutarch.Lift (AsDefaultUni, PBuiltinType, PHaskellType, PLift (..), pconstant)
+import Plutarch.Lift (PLifted, PUnsafeLiftDecl)
 
 -- ctor in-scope for deriving
 import Plutarch.Prelude
-import Plutarch.Tuple (PTuple)
 import qualified Plutus.V1.Ledger.Api as Plutus
-import qualified Plutus.V1.Ledger.Crypto as Plutus
-import qualified PlutusTx.AssocMap as AssocMap
-import PlutusTx.Builtins.Internal (BuiltinByteString (..))
 import qualified PlutusTx.Prelude as PlutusTx
 
 --------------------------------------------------------------------------------
 
+type PTuple = PDataList
```

Comment:
Good point, but this does remove lifting to Haskell tuples as a side-effect :thinking: